app/views/articles/delete_confirm.html.erb

```diff
@@ -17,7 +17,7 @@
       <a data-no-instant href="<%= @article.path %>/edit" class="crayons-btn crayons-btn--secondary">
         <%= @article.published ? "Unpublish" : "Edit" %>
       </a>
-      <a data-no-instant href="<%= @article.path %>" class="crayons-btn crayons-btn--ghost">Dismiss</a>
+      <a data-no-instant href="/dashboard" class="crayons-btn crayons-btn--ghost">Dismiss</a>
```

Comment:
On second thought, it might make more sense to navigate `back` but I think `/dashboard` is fine too.

Reply:
```suggestion
      <a data-no-instant href=<%= dashboard_path %>" class="crayons-btn crayons-btn--ghost">Dismiss</a>
```

Reply:
Thanks, committed!